docs: add note about production environment considerations for tests (#1704)
shopware/docs: guides/hosting/performance/k6.md

## Running the tests

> ::: info

Reviewer: ```suggestion
::: warning
```

> When running against a production environment, make sure you have informed your hosting provider as maybe your IP will be blocked for a short time or limited to a certain amount of requests, which could lead to a false positive. Generally Grafana Cloud is recommended as it allows to distribute the load across multiple locations.

Reviewer: ```suggestion
When running against a production environment, make sure you have informed your hosting provider. Your IP may be blocked for a short time or limited to a certain number of requests, which could lead to a false positive. Grafana Cloud is generally recommended as it allows you to distribute the load across multiple locations.
```
